src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @next/next/no-img-element */
+import styles from '@/components/leaderboard/TopThreeCard/style.module.scss';
```

Comment:
```suggestion
import styles from './style.module.scss';
```
Since this file exists in the same file, we can import with a relative path instead of an absolute path